8.0/root/usr/libexec/container-setup

```diff
@@ -67,6 +67,8 @@ if [ "x$PLATFORM" == "xel9" ] || [ "x$PLATFORM" == "xfedora" ]; then
   fi
   if [ "${PHP_CLEAR_ENV:-false}" == "true" ]; then
     echo "clear_env = yes" >> "${PHP_FPM_CONF_D_PATH}/${PHP_FPM_CONF_FILE}"
+  else
+    sed -e 's/^(clear_env)\s+.*/clear_env = no/' -i ${PHP_FPM_CONF_D_PATH}/${PHP_FPM_CONF_FILE}
```

Comment:
Just wondering - why do we explicitly check for existing `clear_env`  definitions here and not in the `true` case?